app/_data/es.yml

```diff
@@ -322,6 +322,11 @@ about:
     text1:      "La adhesión al proyecto está abierta a cualquier ONG, institución de educación o grupo de la sociedad civil dispuestos a contribuir a los objetivos y cumplir con la ética expuesta anteriormente. La aprobación de la adhesión es responsabilidad de las organizaciones actualmente afiliadas."
     text2:      "Si estás interesado en adherirse como miembro, por favor ponte en contacto con nosotros mediante el siguiente formulario."
   contact_btn:  "CONTÁCTANOS PARA CONVERTIRTE EN MIEMBRO"
+   communities:
```

Comment:
Tiny issue here, the indentation is incorrect. Should be at the same level as the previous line